index.html

```diff
@@ -17,10 +17,49 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container column-full header">Code Journal</div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <form>
+          <div class="row">
+            <div class="column-full">
+              <h1>New Entry</h1>
+            </div>
+            <div class="column-half">
+              <img
+                src="images/placeholder-image-square.jpg"
+                alt="placeholder-image" />
+            </div>
+            <div class="column-half">
+              <div class="label-box">
+                <label for="title">Title</label>
+              </div>
+              <div class="input-box">
+                <input type="text" id="title" name="title" required />
+              </div>
+              <div class="label-box">
+                <label for="photo-url">Photo URL</label>
+              </div>
+              <div class="input-box">
+                <input type="text" id="photo-url" name="photo-url" required />
+              </div>
+            </div>
+            <div class="column-full label-box">
+              <label for="notes">Notes</label>
+            </div>
+            <div class="column-full textarea-box">
+              <textarea id="notes" name="notes"></textarea>
+            </div>
+            <div class="column-full">
+              <label for="save-button"></label>
+            </div>
```

Comment:
This `label` isn't needed. We only need to include `label`s when there's text inside of it associated with an `input` or `textarea`, etc. Like how we have "Title" above the title `input`

Reply:
i just thought it was good practice due to screen readers

Reply:
It is, but only if there's text inside of it. With no text, there's nothing for the screen reader to read out